src/utils/display_charts.py

```diff
@@ -54,3 +54,33 @@ def display_line_chart(game_id):
 
 def display_table(game_id):
     st.success("Table")
+    res_dict = requests.request(
+        "GET", f"https://www.cheapshark.com/api/1.0/games?id={game_id}"
+    ).json()
+    
+    deals_df = pd.DataFrame(res_dict['deals'])
+    
+    #potentially nuke this next line just incase
```

Comment:
Please add spaces after the `#`,

Or just enable format on save, you can find this in `settings -> search (actions on save) -> enable: Reformat code, Optimize Imports, Run Black